appservice/src/intTest/groovy/nu/yona/server/DeviceManagementTest.groovy

```diff
@@ -139,7 +141,7 @@ class DeviceManagementTest extends AbstractAppServiceIntegrationTest
 		when:
 		def newDeviceName = "My iPhone"
 		def newDeviceOs = "ANDROID"
-		def response = appService.registerNewDevice(registerUrl, "Wrong password", newDeviceName, newDeviceOs, "0.1")
+		def response = appService.registerNewDevice(registerUrl, "Wrong password", newDeviceName, newDeviceOs, "0.1", null)
```

Comment:
Make the Firebase device ID an optional parameter, so not everyone needs to bother.

Reply:
OK, good point